app/tests/test_admin_page.py

```diff
@@ -342,6 +342,145 @@ def test_add_update_collection_with_provider(self):
             msg="Admin page wrong provider.roles after update"
         )
 
+    def test_add_collection_with_provider_no_description(self):
+        # Login the user first
+        self.client.login(username=self.username, password=self.password)
+
+        data = {
+            "name": "test_collection",
+            "license": "free",
+            "description": "some very important collection",
+            "providers-TOTAL_FORMS": "0",
+            "providers-INITIAL_FORMS": "0",
+            "links-TOTAL_FORMS": "0",
+            "links-INITIAL_FORMS": "0"
+        }
+        data.update({
```

Comment:
From the admin interface, no provider description returns a `"providers-0-description": ""` in the payload.

Reply:
I don't exactly understand the problem here.

Reply:
Sorry was not very clear, I mean the admin interface client will always send a `providers-x-description` either with a value or with an empty string. So testing with a payload that don't contains `providers-x-description` doesn't make sense.